### ISSUE TITLE
Configuring routeAfterAuthentication in 3.1.0 without mixins

### ISSUE THREAD

**Issue by chbonser** [simplabs/ember-simple-auth#2240]
When replacing the ESA mixins with the new 3.1.0 features, how do you configure `routeAfterAuthentication`?

**Comment by ballPointPenguin** [comment]
curious about this too

**Comment by ballPointPenguin** [comment]
This is working for me now. I'm not sure if it's the "correct" approach:

```js
import ESASessionService from 'ember-simple-auth/services/session'
import { inject as service } from '@ember/service'

export default class SessionService extends ESASessionService {
  @service currentUser

  handleAuthentication() {
    super.handleAuthentication('profile')

    this.currentUser.load().catch(e => {
      console.error(e)
      this.invalidate()
    })
  }
}
```

**Comment by ballPointPenguin** [comment]
I'm overwriting the default 'index' route with 'profile'.

**Comment by marcoow** [comment]
@ballPointPenguin's approach is indeed the correct and suggested approach.

**Comment by chbonser** [comment]
I ended up overwriting the default 'index' route as well. I do think it would be worth a note in the change log / upgrade guide on this topic as I expect others will have the same question.

**Comment by rjschie** [comment]
I believe the way to do this is is the same as pre-3.1.0:  inside the config environment. However, there was a bug with the lookup being used to fetch that config which isn't yet released in the 3.1.0 beta tags (https://github.com/simplabs/ember-simple-auth/issues/2231). Until that gets released you have to double-nest the config. I'd recommend doing both so you're ready for the upgrade, and able to use the workaround for now:

```js
// config/environment.js
// ...

'ember-simple-auth': {
  routeAfterAuthentcation: 'profile',
  'ember-simple-auth': {
    routeAfterAuthentcation: 'profile',
  },
}
```

**Comment by MichalBryxi** [comment]
For anyone wondering: In `ember-simple-auth 4.2.2` one of the working ways to get similar functionality as we have in `routeAfterAuthentication` is:

```js
// app/session/service.js 

import Service from 'ember-simple-auth/services/session';

export default class SessionService extends Service {
  handleAuthentication() {
    super.handleAuthentication('authenticated.invoices'); // your redirect
  }
}
```
